### PR TITLE
Update BMLor321

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I updated the record according to Marrassinis catalogue.

### CODE REVIEW

File: FlorenceBML/BMLor321/BMLor321.xml [hunk 1]
```diff
@@ -34,8 +34,133 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                        <altIdentifier>
+                            <idno>Assemani CCCCV</idno>
+                        </altIdentifier>
                     </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1r" to="68v"/>
+                            <title>Gǝʿǝz-Latin Lexicon</title>
+                            <note>According to <bibl><ptr target="bm:bm:Assemani1719Bibliotheca"/><citedRange unit="page">431</citedRange>
```
Comment (thea-m) [inline FlorenceBML/BMLor321/BMLor321.xml:46]: double bm:

File: FlorenceBML/BMLor321/BMLor321.xml [hunk 2]
```diff
@@ -34,8 +34,133 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                </bibl> the author is <persName ref="PRS14230GBRaimondi"/>.
+                                The Vocabulary is apparently taken from lyrics and arranged by initial letter, but many pages remain blank, i.e. the work is
```
Comment (thea-m) [inline FlorenceBML/BMLor321/BMLor321.xml:48]: I don't understand what "taken from lyrics" means

File: FlorenceBML/BMLor321/BMLor321.xml [hunk 3]
```diff
@@ -34,8 +34,133 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                     </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1r" to="68v"/>
+                            <title>Gǝʿǝz-Latin Lexicon</title>
+                            <note>According to <bibl><ptr target="bm:bm:Assemani1719Bibliotheca"/><citedRange unit="page">431</citedRange>
+                                </bibl> the author is <persName ref="PRS14230GBRaimondi"/>.
+                                The Vocabulary is apparently taken from lyrics and arranged by initial letter, but many pages remain blank, i.e. the work is 
+                                largely unfinished. Many Ethiopian words are given in a status construct, causatives or brokan plural. The succession is
```
Comment (thea-m) [inline FlorenceBML/BMLor321/BMLor321.xml:49]: "in construct state"?

File: FlorenceBML/BMLor321/BMLor321.xml [hunk 4]
```diff
@@ -34,8 +34,133 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1r" to="68v"/>
+                            <title>Gǝʿǝz-Latin Lexicon</title>
+                            <note>According to <bibl><ptr target="bm:bm:Assemani1719Bibliotheca"/><citedRange unit="page">431</citedRange>
+                                </bibl> the author is <persName ref="PRS14230GBRaimondi"/>.
+                                The Vocabulary is apparently taken from lyrics and arranged by initial letter, but many pages remain blank, i.e. the work is 
+                                largely unfinished. Many Ethiopian words are given in a status construct, causatives or brokan plural. The succession is 
+                                different from the traditional order and may be seen as a modified ʾabugidā-type: <foreign xml:lang="gez">አ፡ በ፡ ገ፡ ደ፡ ሀ፡ ወ፡
```
Comment (thea-m) [inline FlorenceBML/BMLor321/BMLor321.xml:50]: broken

File: FlorenceBML/BMLor321/BMLor321.xml [hunk 5]
```diff
@@ -34,8 +34,133 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material cert="medium" key="paper"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf" quantity="134">189</measure>
```
Comment (thea-m) [inline FlorenceBML/BMLor321/BMLor321.xml:63]: could you explain the difference between 134 and 189?

File: FlorenceBML/BMLor321/BMLor321.xml [hunk 6]
```diff
@@ -34,8 +34,133 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                </item>
+                                <item xml:id="e6">
+                                    <desc type="StampExlibris"><foreign xml:lang="it">Dell'aff.mo sig. Cardinale di S. Giorgio. Fu del fu fra Thomaso</foreign>
+                                        in ink by a hand that is not recent, on a sheet of paper glued to the inside front cover.</desc>
+                                </item>
+                                <item xml:id="e7">
+                                    <desc type="StampExlibris"><foreign xml:lang="la">egyptio</foreign> in ink at the front cover.</desc>
+                                </item>
+                                <item xml:id="e8">
+                                    <desc type="StampExlibris">'402' in red ink at a paper batch glued at the front cover and corrected to '405'.</desc>
```
Comment (thea-m) [inline FlorenceBML/BMLor321/BMLor321.xml:136]: different quotation marks from those used for the shelfmarks above

**Review by thea-m — CHANGES_REQUESTED** [review]
Nice manuscript, thank you!

File: FlorenceBML/BMLor321/BMLor321.xml [hunk 7]
```diff
@@ -34,8 +34,133 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                         </altIdentifier>
+                        <altIdentifier>
+                            <idno>Assemani CCCCV</idno>
+                        </altIdentifier>
                     </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1r" to="68v"/>
+                            <title>Gǝʿǝz-Latin Lexicon</title>
```
Comment (eu-genia) [inline FlorenceBML/BMLor321/BMLor321.xml:45]: I think you need to create an ID, of the type Studies

File: FlorenceBML/BMLor321/BMLor321.xml (hunk 1 above)
Comment (eu-genia) [inline FlorenceBML/BMLor321/BMLor321.xml:46]: this note belongs to the text (STU0008Lex or something) not the manuscript

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: FlorenceBML/BMLor321/BMLor321.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLor321/BMLor321.xml:48]: I rendered to "poetry".

File: FlorenceBML/BMLor321/BMLor321.xml (hunk 3 above)
Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLor321/BMLor321.xml:49]: Yes.

File: FlorenceBML/BMLor321/BMLor321.xml (hunk 5 above)
Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLor321/BMLor321.xml:63]: Copy-paste error - it is 189 in fact.

File: FlorenceBML/BMLor321/BMLor321.xml (hunk 2 above)
Comment (eu-genia) [reply FlorenceBML/BMLor321/BMLor321.xml:48]: It is still not clear (lemma definitions are rhymed?)

In any case this description is the description of work (study) record to be created, not of the manuscript (even if a work is contained in a unique copy, we still must keep content and carrier apart)

File: FlorenceBML/BMLor321/BMLor321.xml [hunk 8]
```diff
@@ -34,8 +34,133 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                        <locus from="53" to="55"/>
+                                        <locus target="#57"/>
+                                        <locus from="59" to="65"/>
+                                        <locus from="69" to="188"/>
+                                    <dimensions type="outer" unit="mm"></dimensions>
+                                        <height cert="medium">240</height>
+                                        <width cert="medium">139</width>
+                                        <height cert="medium">31</height>
+                                </extent>
+                                <foliation>Foliated in pen by a recent European hand at the upper right by the author's own hand ending up with 188,
```
[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLor321/BMLor321.xml:94]: Yes, you are right. It is redundant.

File: FlorenceBML/BMLor321/BMLor321.xml (hunk 7 above)
Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLor321/BMLor321.xml:45]: OK. I expect this will be very similar to a work IK.

Comment (eu-genia) [reply FlorenceBML/BMLor321/BMLor321.xml:45]: Yes it is the same as work but gets a different prefix (STU not LIT).

File: FlorenceBML/BMLor321/BMLor321.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLor321/BMLor321.xml:46]: Alright! I created a STU record with that ID and removed the information to that record.

File: FlorenceBML/BMLor321/BMLor321.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLor321/BMLor321.xml]: Yes, I have removed this information to the STU record. It is not, as you expected a lexicon in a rhymed form, but Marrassini (or Assemani) suspected, that the lemmata are taken from poetic texts.

**Review by thea-m — APPROVED** [review]

**Pull request merged**